mllib/src/main/scala/org/apache/spark/ml/r/LDAWrapper.scala

```diff
@@ -172,6 +187,8 @@ private[r] object LDAWrapper extends MLReadable[LDAWrapper] {
       model,
       ldaModel.logLikelihood(preprocessedData),
       ldaModel.logPerplexity(preprocessedData),
+      trainingLogLikelihood,
+      logPrior,
```

Comment:
The first version, I got them from the model in the `LDAWrapper` class. However, when I read `logPrior`, I found that the loaded `logPrior` is not the same as the value before save. So, I followed the logLikelihood and logPerplexity to save it in the metadata.